src/index.js

```diff
@@ -0,0 +1,23 @@
+import CSpice from './cspice.js';
```

Comment:
Can we just `export` these files directly? Instead of `importing` first and then `exporting`? For example:
```
export * as CSpice from './cspice.js';
export * as Spice from './spice.js';
export {
    chronos,
    loadKernel,
    loadKernelFromBuffer,
    parseMetakernel,
    prepareFileFromBuffer,
    removeFile,
    unloadKernel
} from './timecraft.js';
```

Reply:
I don't think that valid in es6 modules is it?

Reply:
https://stackoverflow.com/questions/38077164/es6-export-from-import

🤯

I'll try making the change.

Reply:
Done!